### PR TITLE
Add header to match figma DL

### LINKED ISSUE #91: Add Header to Vocabulary
Blocked by https://github.com/creativecommons/vocabulary/issues/100 where we just need to wrap up language changes to the Header.

We need to add the Header to Vocabulary to match the Figma DL: https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=179%3A0

### PR DESCRIPTION
**Pull request by chidexebere**
**Fixes**

Fixes #91 by @annatuma and #149 by @panchovm 

**Description**
This adds the Header (Big and Small) to Vocabulary to match the Figma DL

**Type of PR** 
This PR is a feature.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

File: src/stories/header.stories.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { CCLogomark } from './logos.stories'
+
+export default { title: 'Header' }
+
+export const bigHeader = () =>
+    `<header class="header big">
+        <div class="container">
+            <a href="#" class="explore-tab">Explore CC</a>
```
Comment (hugosolar) [inline src/stories/header.stories.js]: Hey @chidexebere the explore tab should be part of the global navigation. so there's no need to be in this component

File: src/stories/header.stories.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                </div>
+            </nav>
+        </div>
+    </header>
+`
+
+export const smallHeader = () =>
+    `<header class="header small">
+        <div class="container">
+            <a href="#" class="explore-tab">Explore CC</a>
```
Comment (hugosolar) [inline src/stories/header.stories.js]: Same here. The explore tab is part of the global navigation component

File: src/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  &.small {
+    padding: 1.25rem 5.8rem;
+  }
+
+  .container {
+    .explore-tab {
+      position: absolute;
+      z-index: 1000;
+      padding: 0.5rem 1rem;
```
Comment (hugosolar) [inline src/styles/header.scss]: There are some `spaces` variables in `spaces.scss` we should use those variables

**Review by hugosolar — CHANGES_REQUESTED** [review]

File: src/styles/header.scss (hunk 3 above)
Comment (chidexebere) [reply src/styles/header.scss]: Made use of spaces variables where they apply

File: src/stories/header.stories.js (hunk 2 above)
Comment (chidexebere) [reply src/stories/header.stories.js]: Removed

File: src/stories/header.stories.js (hunk 1 above)
Comment (chidexebere) [reply src/stories/header.stories.js]: Removed

**Review by hugosolar — CHANGES_REQUESTED** [review]
Last detail. The hover state of the navigation element should be as in figma. it's just a small color change of the selected element and the background doesn't change

Screen Shot 2020-02-27 at 9 35 08 PM

Also, the arrow is not the same as the figma design.

**Comment by chidexebere** [comment]
> Last detail. The hover state of the navigation element should be as in figma. it's just a small color change of the selected element and the background doesn't change
> 
> Screen Shot 2020-02-27 at 9 35 08 PM
> 
> Also, the arrow is not the same as the figma design.

done

**Comment by chidexebere** [comment]
> Last detail. The hover state of the navigation element should be as in figma. it's just a small color change of the selected element and the background doesn't change
> 
> Screen Shot 2020-02-27 at 9 35 08 PM
> 
> Also, the arrow is not the same as the figma design.

fixed

File: src/styles/header.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  &.small {
+    padding: 1.25rem 5.8rem;
+  }
+
+  .navbar {
+    background-color: $color-white;
+
+    .navbar-item:hover,
+    .navbar-link:hover,
+    .navbar-link:visited {
```
Comment (hugosolar) [inline src/styles/header.scss:17]: You also need to specify the color of the text... there's a slight color change from dark-slate-gray to black as you can see in the image

Screen Shot 2020-02-27 at 9 35 08 PM

**Review by hugosolar — CHANGES_REQUESTED** [review]

File: src/styles/header.scss (hunk 4 above)
Comment (chidexebere) [reply src/styles/header.scss:17]: fixed....its actually from light-slate-gray to black

**Review by hugosolar — APPROVED** [review]

**Pull request merged**